paper_figures/code/maxsavings_boxplot.py

```diff
@@ -63,18 +65,24 @@
         )
 
         # Tariff
-        tariff = ps.gettariff(region=reg)
-        startdate_dt, enddate_dt = ms.get_start_end(month)
-        month_length = int((enddate_dt - startdate_dt) / np.timedelta64(1, "h"))
-        tariff_savings_sweep[i, j] = ms.max_tariff_savings(
-            data=tariff,
-            system_uptime=0.0,
-            continuous_flex=0.0,
-            baseload=np.ones(month_length),
-            startdate_dt=startdate_dt,
-            enddate_dt=enddate_dt,
-            uptime_equality=False
-        )
+        tariffs = ps.gettariff(region=reg, full_list=True)
+        for tariff in tariffs:
+            startdate_dt, enddate_dt = ms.get_start_end(month)
+            month_length = int((enddate_dt - startdate_dt) / np.timedelta64(1, "h"))
+            tariff_savings = ms.max_tariff_savings(
+                data=tariff,
+                system_uptime=0.0,
+                continuous_flex=0.0,
```

Comment:
Should continuous flex here be 0 or 1 for the tariff case? I'm thinking we should make it 1 if we're looking at max savings.